src/components/GeneralInfo/GeneralInformation/general-information.scss

```
@@ -56,4 +56,18 @@
         white-space: normal;
     }
     }
+
+    .ins-c__no-border {
+        border: none;
+    }
+    
+    .ins-c-data-collectors__expaded-row {
+        padding-top: 10px;
+        padding-bottom: 15px;
+        padding-left: 15px;
```

Instead of `px` values we should use `rem` and we should also avoid using direct values in favor of css vars
```suggestion
        padding-top: var(--pf-global--spacer--sm);
        padding-bottom: var(--pf-global--spacer--md);
        padding-left: var(--pf-global--spacer--md);
```